src/plugins/dam/components/asset-snippet/Image.jsx

```diff
@@ -23,17 +23,20 @@ const Image = ({ asset, previewUrl }) => (
       </FormGroup>
     </div>
     <div className="col-12 col-md-6">
+      {!previewUrl && <Spinner centered className="m-0 mr-2" color="#08a0e8" width="24" strokeWidth="8" />}
```

Comment:
@jalopez13 remove custom color here.  if a style is needed it should come from bootstrap or admin-ui.  primary, secondary, warning, danger, etc. vs a hard coded color.